waterbury/the-happy-christian/book.txt

```diff
@@ -317,72 +317,68 @@ We will not undertake to say, how many professors of piety are thus drawn away b
 
 The lamentation of Demas, with which the last chapter concludes, implies a fault too common among professing Christians; especially those whose business and engagements lead them into frequent contact with the world. This obstruction to their piety, and of course to their true felicity and joy, is great in proportion to the time consumed and the interest felt in earthly pleasures and connections.
 
-We do not admit the impossibility of mingling with the world and still retaining our peace of mind, our Christian influence, and our pious joy. Many might be named, who keep " the garment unspotted," and the soul unclogged amid the cares of earth, and under the pressure of its daily toil. It would be an argument against our religion, if it disqualified its possessor for the performance of any duty, social, civil or political; if it did not, in fact, fit him the better to discharge these obligations. It is in accordance with the spirit of Christianity to meet cheerfully every occasion which Providence furnishes for the promotion of the general good. It is the duty of her professors to shrink from no burden which may lawfully be borne, and to retreat from no station—if personally qualified—in which they may serve their country without dishonoring their religion. "Faith overcomes the world." This is the testimony of Heaven. But this victory implies not a retreat from, but a conflict with, the foe. We are to pray, not to be "taken out of the world; but to be kept from the evil."
+We do not admit the impossibility of mingling with the world and still retaining our peace of mind, our Christian influence, and our pious joy. Many might be named, who keep "the garment unspotted," and the soul unclogged amid the cares of earth, and under the pressure of its daily toil. It would be an argument against our religion, if it disqualified its possessor for the performance of any duty, social, civil or political; if it did not, in fact, fit him the better to discharge these obligations. It is in accordance with the spirit of Christianity to meet cheerfully every occasion which Providence furnishes for the promotion of the general good. It is the duty of her professors to shrink from no burden which may lawfully be borne, and to retreat from no station---  if personally qualified --- in which they may serve their country without dishonoring their religion. "Faith overcomes the world." This is the testimony of Heaven. But this victory implies not a retreat from, but a conflict with, the foe. We are to pray, not to be "taken out of the world; but to be kept from the evil."
```

Comment:
Will fix in next edit :) Wasn't sure on the conversion end if spaces were needed to distinguish.

Lukewil <notifications@github.com> wrote:
>In waterbury/the-happy-christian/book.txt:
>
>> @@ -317,72 +317,68 @@ We will not undertake to say, how many professors of piety are thus drawn away b > > The lamentation of Demas, with which the last chapter concludes, implies a fault too common among professing Christians; especially those whose business and engagements lead them into frequent contact with the world. This obstruction to their piety, and of course to their true felicity and joy, is great in proportion to the time consumed and the interest felt in earthly pleasures and connections. > > -We do not admit the impossibility of mingling with the world and still retaining our peace of mind, our Christian influence, and our pious joy. Many might be named, who keep " the garment unspotted," and the soul unclogged amid the cares of earth, and under the pressure of its daily toil. It would be an argument against our religion, if it disqualified its possessor for the performance of any duty, social, civil or political; if it did not, in fact, fit him the better to discharge these obligations. It is in accordance with the spirit of Christianity to meet cheerfully every occasion which Providence furnishes for the promotion of the general good. It is the duty of her professors to shrink from no burden which may lawfully be borne, and to retreat from no station—if personally qualified—in which they may serve their country without dishonoring their religion. "Faith overcomes the world." This is the testimony of Heaven. But this victory implies not a retreat from, but a conflict with, the foe. We are to pray, not to be "taken out of the world; but to be kept from the evil." > +We do not admit the impossibility of mingling with the world and still retaining our peace of mind, our Christian influence, and our pious joy. Many might be named, who keep "the garment unspotted," and the soul unclogged amid the cares of earth, and under the pressure of its daily toil. It would be an argument against our religion, if it disqualified its possessor for the performance of any duty, social, civil or political; if it did not, in fact, fit him the better to discharge these obligations. It is in accordance with the spirit of Christianity to meet cheerfully every occasion which Providence furnishes for the promotion of the general good. It is the duty of her professors to shrink from no burden which may lawfully be borne, and to retreat from no station--- if personally qualified --- in which they may serve their country without dishonoring their religion. "Faith overcomes the world." This is the testimony of Heaven. But this victory implies not a retreat from, but a conflict with, the foe. We are to pray, not to be "taken out of the world; but to be kept from the evil." 
>
>Just wanted to make a note here real quick: em dashes (---) should not have spaces around them, as per Chicago Manual of Style. ￼ 
>
>—
>Reply to this email directly or view it on GitHub.￼
>